python-sdk/tutorials/automl-with-azureml/forecasting-many-models/auto-ml-forecasting-many-models.ipynb

```diff
@@ -750,19 +766,9 @@
     "forecasting_results_name = \"forecasting_results\"\n",
     "forecasting_output_name = \"many_models_inference_output\"\n",
     "forecast_file = get_output_from_mm_pipeline(\n",
-    "    inference_run, forecasting_results_name, forecasting_output_name\n",
+    "    inference_run, forecasting_results_name, forecasting_output_name, output_file_name\n",
     ")\n",
-    "df = pd.read_csv(forecast_file, delimiter=\" \", header=None)\n",
-    "df.columns = [\n",
-    "    \"Week Starting\",\n",
-    "    \"Store\",\n",
-    "    \"Brand\",\n",
-    "    \"Quantity\",\n",
-    "    \"Advert\",\n",
-    "    \"Price\",\n",
-    "    \"Revenue\",\n",
-    "    \"Predicted\",\n",
-    "]\n",
+    "df = pd.read_csv(forecast_file)\n",
```

Comment:
In the past, we read the .txt file that did not contain column names. Is there a separate PR that fixes the issue of reading the output file that contains column names?

Reply:
yes, there is another PR ([this one](https://msdata.visualstudio.com/Vienna/_git/AzureMlCli/pullrequest/825414?_a=overview)) that covers this change. In summary:

- Many Models now provides the capability to generate prediction output in csv format as well.
- Many Models prediction will now include **column names** in the output file in case of **csv file format**.